PWGJE/Tasks/photonChargedTriggerCorrelation.cxx

```diff
@@ -0,0 +1,1725 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file photonChargedTriggerCorrelation.cxx
+/// \author Julius Kinner
+/// \brief photon-jet correlation analysis
+///
+/// Analysis for angular correlations between jets and photons via two-particle correlations with charged high-pt triggers
+/// Associated hadrons (tracks), pipm, photons (PCM), pi0 (PCM)
+/// Also contains checks and monte-carlo (efficiency, purity, mc-true correlation,...)
+/// End goal of studying correlations between direct photons and jets
+
+#define ETA_MAX_DEFAULT 0.8
```

Comment:
do we really need to define like this? I think a const double etc in the current namespace would be the better idea